### PR TITLE
final UI for signin and sign up

### PR DESCRIPTION
**Pull request by palrit**

### CODE REVIEW

**Comment by shamreenshaik** [comment]
proceed

**Comment by shailscript** [comment]
@shamreenshaik If you think the changes are good, you should approve the changes from GitHub. Once you approve them, the code owner would merge them back to the main branch i.e., `master` in our project.
If you close it without merging, you'd see these changes on master. So, please let me know when you do a decisive action on a PR and we can work together to get you started.

**Review by shailscript — APPROVED** [review]
Looks good to me, great work! 👍

**Pull request merged**
